lnrpc/routerrpc/router.proto

```diff
@@ -239,6 +239,9 @@ message Failure {
     the failure message. Position zero is the sender node.
     **/
     uint32 failure_source_index = 8;
+
+    /// A failure type-dependent block height.
+    uint32 height = 9;
```

Comment:
perhaps something like `accept_height`, to mimic what the invoice htlc's will show?

Reply:
I thought about that, but wasn't sure. The idea of the `message Failure` is that it isn't a `oneof` to keep it simple. Fields are reused in different messages. This `height` is meant as a generic height field associated with a failure. But yes, I could also make it more specific.